### PR TITLE
feature/radius_audit_log_to_redis

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Use redis to store RADIUS logs.

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## New Features
* Use redis to store RADIUS logs.

### CODE REVIEW

File: go/cron/flush_radius_audit_log_job.go [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+		rows_affected += len(a)
+		jsonStr := "[" + strings.Join(a, ",") + "]"
+		var entries [][]interface{} = make([][]interface{}, len(a))
+		json.Unmarshal([]byte(jsonStr), &entries)
+		j.flushLogs(entries)
+		if time.Now().Sub(start) > j.Timeout {
+			break
+		}
+	}
+	if rows_affected > -1 {
```
Comment (stgmsa) [inline go/cron/flush_radius_audit_log_job.go]: do we need to check this ?

**Review by stgmsa — COMMENTED** [review]
left a change & concern

File: go/cron/flush_radius_audit_log_job.go (hunk 1 above)
Comment (jrouzierinverse) [reply go/cron/flush_radius_audit_log_job.go]: You are correct that check is not useful.

**Review by stgmsa — APPROVED** [review]
looks good to me

**Comment by extrafu** [comment]
@jrouzierinverse Please rebase so this gets merged.

**Pull request merged**